taxi/tests/test_views.py

```diff
@@ -0,0 +1,131 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+from django.urls import reverse
+
+from taxi.models import Manufacturer, Car, Driver
+
+
+class IndexTest(TestCase):
+    @classmethod
+    def setUpTestData(cls):
+        for id_ in range(10):
+            manufacturer = Manufacturer.objects.create(
+                name=f"Brand {id_}",
+                country="Ukraine"
+            )
+
+            Driver.objects.create_user(
+                username=f"driver {id_}",
+                license_number=str(id_)
+            )
+
+            Car.objects.create(
+                model=f"Model {id_}",
+                manufacturer=manufacturer
+            )
+
+    def setUp(self):
+        self.user = get_user_model().objects.create(
+            username="user1",
+            password="top_user",
+        )
+        self.client.force_login(self.user)
+
+    def test_index_counters(self):
+        response = self.client.get(reverse("taxi:index"))
+        self.assertEqual(response.context["num_cars"], 10)
+        self.assertEqual(response.context["num_drivers"], 11)
+        self.assertEqual(response.context["num_manufacturers"], 10)
+
+
+class ManufacturerTest(TestCase):
+    @classmethod
+    def setUpTestData(cls):
+        for id_ in range(10):
+            manufacturer = Manufacturer.objects.create(
+                name=f"Brand {id_}",
+                country="Ukraine"
+            )
+
+            Driver.objects.create_user(
+                username=f"driver {id_}",
+                license_number=str(id_)
+            )
+
+            Car.objects.create(
+                model=f"Model {id_}",
+                manufacturer=manufacturer
+            )
+
+    def setUp(self):
+        self.user = get_user_model().objects.create(
+            username="user1",
+            password="top_user",
+        )
+        self.client.force_login(self.user)
+
+    def test_manufacturer_list_correct_status(self):
+        response = self.client.get(reverse("taxi:manufacturer-list"))
+        self.assertEqual(response.status_code, 200)
+
+    def test_manufacturer_list_pagination(self):
+        response = self.client.get(reverse("taxi:manufacturer-list"))
+        self.assertEqual(response.context["manufacturer_list"].count(), 5)
```

Comment:
do you really need to test this? I guess since it's a built-in option, everything should be fine